addons/web/static/src/js/views/kanban/kanban_renderer_mobile.js

```diff
@@ -300,13 +299,20 @@ KanbanRenderer.include({
         this.activeColumnIndex = moveToIndex;
         var column = this.widgets[this.activeColumnIndex];
         return new Promise(function (resolve) {
-            self.trigger_up('kanban_load_records', {
-                columnID: column.db_id,
-                onSuccess: function () {
-                    self._layoutUpdate(animate);
-                    resolve();
-                },
-            });
+            var processUpdate = function () {
+                self._layoutUpdate(animate);
+                // enable swipe no need to wait the end of the loading contains
+                self._enableSwipe();
+                resolve();
+            };
+            // Check if we need to fetch collapsed column
+            if (column.data.offset === 0 && column.data.count > 0 && column.data.data.length === 0) {
+                self.trigger_up('kanban_load_records', {
```

Comment:
can we try to use _onToggleColumn instead of a specific mobile method _onLoadColumnRecords